src/etc/modsecurity.d/modsecurity-override.conf

```diff
@@ -6,7 +6,6 @@ SecRequestBodyAccess ${MODSEC_REQ_BODY_ACCESS}
 
 SecRequestBodyLimit ${MODSEC_REQ_BODY_LIMIT}
 SecRequestBodyNoFilesLimit ${MODSEC_REQ_BODY_NOFILES_LIMIT}
-SecRequestBodyInMemoryLimit 131072

```

Comment:
Why is this removed?

Reply:
Seems like this is not used anymore in ModSecurity v3.x [1] and the used value (131072) is the default in v2.x anyways [2]. So, good to remove :+1: 

* [1] https://github.com/SpiderLabs/ModSecurity/issues/1516
* [2] https://github.com/SpiderLabs/ModSecurity/wiki/Reference-Manual-(v2.x)#secrequestbodyinmemorylimit